projects/lists/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@skyux/lists",
+  "version": "5.0.0-beta.1",
+  "peerDependencies": {
+    "@angular/common": "^12.2.0",
+    "@angular/core": "^12.2.0",
+    "@angular/platform-browser": "^9.1.6 || ^12",
+    "@skyux/animations": "^5.0.0-beta.0",
```

Comment:
These version number should probably be updated to whatever is in the dependency list in the workspace's `package.json` file since they're all basically broken at this point.